app/metrics.yaml

```diff
@@ -19,3 +19,19 @@ toolbar.events:
       - tlong@mozilla.com
       - aplacitelli@mozilla.com
     expires: 2019-10-01
+experiments.metrics:
+  active_experiment:
+    type: string
+    description: >
+      Records the branch name of the active experiment, if the client is enrolled in any experiments.
```

Comment:
This paragraph reads like this would be true for any experiment. That makes me think that the telemetry is set by the library.

However the code below makes me think that the telemetry is only set if an experiment manually does it. These things seem at odds. Am I misreading something?

Reply:
I re-read this and I'm going to change the wording as it does say 'any experiments' and it should just say the 'reference-browser-test' experiment.

Let me see if I can clarify just to make sure we are still on the same page with how this is implemented:
Yes, the library does record enrollment in any experiment, but the code only records it for the "referenence-browser-test" experiment.  The telemetry that is recorded automatically goes into a section in the Glean ping called 'experiments' and does not require any manual action.  What's going on with the code added to the app is that it is also recording the enrolled branch name manually, to a completely separate metric, if the client matches the experiment "reference-browser-test".  

I was hoping that this satisfied the ideas we discussed in the last meeting about testing both that the client was properly enrolled in the experiment (by looking at the automatically recorded metric) and was able to execute an action on the client as a consequence of the enrollment (in this case the 'action' is a transparent, manually recorded metric in the app code within the `withExperiment` block).